Charts: update 'Discrete Aggregation' demo description (#2511)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/DiscreteAggregation/description.md

To [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) points on discrete axes, follow the steps below:

1. Enable aggregation.    
You can enable data aggregation for the following series:

    - Individual series    
    Assign `true` to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[aggregation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/aggregation/).[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/aggregation/#enabled) property.

    - All series of a specific type  

>     Assign `true` to the `aggregaion.enabled` property belonging to the configuration object of a [specific series type](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/).

Reviewer: ```suggestion
    Assign `true` to the `aggregaion.enabled` property that belongs to the configuration object of a [specific series type](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/).
```

    - All series in the Chart   
    Assign `true` to the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[aggregation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/).[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#enabled) property.    

> The order of this list specifies the priority order of aggregation in the component. For example, you can enable aggregation for all series and disable it for a specific series.

Reviewer: > The order of **this list** specifies the priority order of aggregation in the component.

Of what list do we speak? You indirectly imply the three entries above, I understand. Example is good. Still, I would have preferred a more direct explanation. Something like that (you can come up with something even better than that):

```suggestion
Individual series aggregation settings have priority over common aggregation settings in the component. For example, you can enable aggregation for all series and disable it for a specific series.
```

Thing is that all indirect implications are not good. We should strive to make our texts as clear as possible. I use the following guiding principle in my writing: Every word that has potential for being misunderstood - will certainly be misunderstood. And I recommend you to do the same.

> To [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) points on discrete axes, do the following steps:

Reviewer: ```suggestion
To [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) points on discrete axes, follow the steps below:
```

1. Enable aggregation.    

> You can enable data aggregation for individual series, all series of a specific type, or for all series in the Chart. Assign `true` to the [aggregation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/).[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#enabled) property.

Reviewer: Once I set the `aggregation.enabled` property to `true`, what does it do from the things you listed in the previous sentence? Does this action enable data aggregation for all series in the Chart? It's unclear.

Reviewer: I mark this thread as resolved. Thank you.

In this demo, data aggregation is enabled for all series.

> 2. Specify aggregation [method](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#method) (optional).    

Reviewer: Specify "some", "certain", or "any" aggregation method. In this particular case, we deal with indefiniteness.

```suggestion
2. Specify an aggregation [method](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#method) (optional).    
```
